csrc/multidevice/communication.h

```diff
@@ -218,12 +136,42 @@ case of a local copy)
   - If team is of size 2, the unique non-root have no src buffer and one dst
 buffer
 */
-class SendRecv : public Communication {
+
+class NVF_API Communication : public Expr {
```

Comment:
```suggestion
class Communication : public Expr {
```

https://github.com/NVIDIA/Fuser/blob/main/doc/dev/visibility.md#should-i-mark-my-new-method-or-class-as-nvf_api